script.js

```diff
@@ -346,12 +346,12 @@ function keyPressed() {
     moveLeft()
   }
   if (keyCode === DOWN_ARROW) {
-    //cloakwise
-    rotateBlock(1)
+    //clockwise
```

Comment:
In addition to my comment at the definition of `rotateBlock` about the name of the parameter, another way to make things even more readable than adding comments like this is to define at the top of this file constants for these magic values, like:

```
const CLOCKWISE = true;
const COUNTER_CLOCKWISE = false;
```

then you change the calls to, for instance, `rotateBlock(CLOCKWISE)` and it's perfectly clear with no comment.